blank/README.md

```diff
@@ -19,18 +19,17 @@
 
 - Install packages with `yarn` or `npm install`.
   - If you have native iOS code run `npx pod-install`
-- Run `yarn start` to start the bundler.
+- Run `npx expo start` to start the bundler.
```

Comment:
Doesn't matter but we inject the start script when the user bootstraps with CRNA.

Reply:
good to know! Some examples were one way, some the other, so was trying to pick one and apply it across the board. Knowing that it's always there, makes sense to it back to yarn/ npm, which seems more consistent with other templates.